cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -2253,6 +2273,10 @@ describe('CourseOutlinePage', function() {
                 // This is the response for the subsequent fetch operation.
                 AjaxHelpers.respondWithJson(requests, mockCustomPacingServerValuesJson);
 
+                expect($('.outline-subsection .status-custom-grading-date').text().trim()).toEqual(
+                    'Custom due date: 3 weeks from enrollment'
+                );
```

Comment:
Hmmm. Is this okay? Looking at the selector, you're only using classes which means multiple of these could show up on the page. You just happen to only be setting the value on one of them, but I wonder if it would cause any issues if there were multiple.

Reply:
Should these have the same unique locator then? I believe the selector as you have it now would match all `status-custom-grading-date`s in all `outline-subsection`s